---
layout: page
title: "CV"
header-img: "img/yellowstone.jpg"
---

My full CV can be found here <a href="/docs/CV_vshastry-2.pdf" target="_blank"><i class="fa fa-file-text fa-md"></i></a>, and a short form is below.   

___

### Education  

PhD, Committe on Genetics, Genomics and Systems Biology, University of Chicago 2020-present  
Advisor: [John Novembre](http://jnpopgen.org)  

MS, Botany, University of Wyoming 2018-2020
Advisor: [Alex Buerkle](http://uwyo.edu/buerkle)

BS, Electrical Engineering, University of Wyoming 2014-2017
Minor in Computational Science
Advisor: [Suresh Muknahallipatna](http://www.uwyo.edu/electrical/faculty-staff/suresh-muknahallipatna/)

### Publications

Harrison JG, Calder WJ, __Shastry V__, Buerkle CA (2019)
Dirichlet‚Äêmultinomial modelling outperforms alternatives for analysis of microbiome and other ecological count data (published in Molecular Ecology Resources, [bioRxiv](https://www.biorxiv.org/content/10.1101/711317v3))

__Shastry V__, Adams PE, Lindtke D, Mandeville EG, Parchman TL, Gompert Z, Buerkle CA (2020)
Model-based genotype and ancestry estimation for potential hybrids with mixed-ploidy (in review at Molecular Ecology Resources, [bioRxiv](https://www.biorxiv.org/content/10.1101/2020.07.31.231514v1))

### Software

[entropy](https://bitbucket.org/buerklelab/mixedploidy-entropy/src/master/): genotype and ancestry estimation for mixed-ploidy, installable through [source](https://bitbucket.org/buerklelab/mixedploidy-entropy/src/master/) or as a [conda package](https://anaconda.org/bioconda/popgen-entropy)
[CNVRG](https://rdrr.io/github/JHarrisonEcoEvo/CNVRG/): R package for a Dirichlet-Multinomal model to quantify differential abundance in microbial count data
