---
layout: page
title: "Research"
description: "This is what I do"
header-img: "img/screen.jpg"
---

### Evolutionary Genomics of Major Transitions

The majority of my research focuses on how genomes evolve after they transition in ploidy or in mating system, i.e. from diploidy to polyploidy or outcrossing to selfing.
Major transitions on their own have been likened to sudden, large effect mutations, but this is a misleading analogy for several reasons.
These can be understood by the fact that in both transitions, the entire genome is suddenly exposed to different selection pressures, as everything in the genome now has a new fitness level, including genes, regulatory pathways, and genome architecture.
In addition to this new genomic environment to which and within which these species must now adapt, major transitions like these are also expected to result in shifts in major population genetic parameters, including mutation, recombination, selection, and gene flow, that determine how evolution proceeds.
In light of these parameter shifts, the transition to selfing and diploidization effectively rewrite the rules of their evolution.

Despite this, there are major gaps in our empirical understanding of how the mutation rate, the recombination rate and landscape, the strength and dynamics of selection, and their interactions evolve in selfers and polyploids.
My research touches on all of these topics, with a particular focus on parameterizing these rates, and understanding their interactions in the forms of linked selection and introgression.

### Recombination Evolution

<div class="left_figure"><figure>
	<img src="/img/cap2.jpg" alt="Capsella" style="float: left" width="250" hspace="25" vspace="25" />
	<figcaption>
		<font size="2"><i>Capsella bursa-pastoris</i></font>
	</figcaption>
</figure></div>

Meiotic recombination is a major population genetic parameter that determines the evolutionary dynamics in genomes, and has been described as the primary source shaping evolution (Stebbins 1950).
Recombination tends to localize to certain genomic regions, which tend to be upstream of genes in regions of open chromatin in plants, where the average recombination rate is many times higher than the background rate.
Despite being a fundmental source of variation, we know surprisingly little, especially in plants, about how and why rates of recombination vary within and between species.
Additionally, recombination rates have been theorized to evolve differently in species with contrasting mating systems, and the genomic landscape of recombination may evolve in response to other genomic features like transposable elements.
I'm interested in how recombination rates evolve and the factors that determine their genomic distribution within and among species, particularly between selfers and outcrossers and between diploids and polyploids.

Relevant papers: [Kent, Uzunovic, and Wright 2017 _Phil Trans_](https://doi.org/10.1098/rstb.2016.0458)

___

### Linked Selection
<div class="right_figure"><figure>
        <img src="/img/B.gif" alt="linked" style="float: right" width="300" hspace="25" vspace="25" />
        <figcaption>
                <font size="2">background selection effects with increasingly weaker selection</font>
        </figcaption>
</figure></div>

When genomic sites undergo positive or negative selection, the neutral sites surrounding them experience linked selection (known as selective sweeps and background selection, respectively).
This results in a loss of neutral diversity near selected sites, with impacts depending on the strength of selection and the local recombination rate.
Although many studies have investigated either selective sweeps or background selection, it has been difficult to investigate their joint impacts.
Additionally, in species with reduced recombination or efficacy of selection, the effects of linked selection can become more complicated, as selected sites begin to interfere with each other, and in polyploids, genomic redundancy could lead to lower or higher impacts of linked selection.

___

### Introgression

<div class="left_figure"><figure>
        <img src="/img/rice.png" alt="Rice" style="float: left" width="450" hspace="25" vspace="25" />
        <figcaption>
                <font size="2">Nucleotide diversity along a chromosome of rice</font>
        </figcaption>
</figure></div>

I am generally interested in how gene flow and introgression influence and prevent adaptation, and in what shapes the genomic landscape of admixed ancestry.
Admixed ancestry has the potential to be positively selected and influence the adaptation of species or act as deleterious mutational input that reinforces species boundaries.
These opposing forces of selection, in combination with normally-occuring genomic processes like background selection can create similar patterns that are difficult to distinguish in genomic data.
I have worked on projects investigating the role of introgression in evolution in selfing and outcrossing _Capsella_ species pairs, diploid and polyploid _Capsella_ species pairs, and wild and domesticated _Oryza_ species pairs.


Relevant papers: [Sas et al. 2016 _Current Biology_ ](https://doi.org/10.1016/j.cub.2016.10.026)

[Kryvokhyzha and Salcedo et al 2018 _bioRxiv_ ](https://doi.org/10.1101/234096)
